componentTests/MigrationTest.cpp

```diff
@@ -248,7 +247,7 @@ SUITE(MigrationTest)
 
             Stopwatch watch;
             watch.Start();
-            for(int i = 0; i < 100000 * 365 * 5; i++ ) // 100k individual female vectors every day for five years
+            for(int i = 0; i < 1000 * 365 * 5; i++ ) // 1k individual female vectors every day for five years
```

Comment:
Thank you for making this smaller.  You might want to create a variable, say num_samples that is equal to this and used in both for-loops. It would ensure they loop the same number of times.